### DISCUSSION TITLE
Toast messages are not shown

### DISCUSSION THREAD

**Discussion by Rudkovskiy427** [orchidsoftware/platform#2823] [Q&A]
There is my screen method, but the Toast message doesn't show (but in older project Toast messages work normally)

    public function save(Model $model, ModelRequest $request)
    {
        $request->validated();

        $model->fill($request->get('model'))->save();

        Toast::success('Success!');

        return redirect()->route('level.index');
    }
    
What could be the reason?

**Comment by Rudkovskiy427** [comment]
I tried to update Orchid from `^14.0.3[14.15.0]` to `^14.0.3[14.21.1]`, but Toast still doesn't work

**Comment by esmultiservicios** [comment]
Are you importing the resource

use Orchid\Support\Facades\Toast;

and then to use it
Toast::info(__('Cliente fue creado correctamente.'));
